docs/content/drafts/DropdownMenu2.mdx

```diff
@@ -0,0 +1,308 @@
+        {duration} {duration > 1 ? 'weeks' : 'week'}
+      </DropdownMenu.Button>
+      <DropdownMenu.Overlay width="medium">
+        <ActionList>
+          {[1, 2, 3, 4, 5, 6].map(weeks => (
+            <ActionList.Item key={weeks} selected={duration === weeks} onSelect={() => setDuration(weeks)}>
+              {weeks} {weeks > 1 ? 'weeks' : 'week'}
+            </ActionList.Item>
+          ))}
+        </ActionList>
+      </DropdownMenu.Overlay>
+    </DropdownMenu>
+  )
+}
+
+render(<Example />)
+```
+
+### With External Anchor
+
+To create an anchor outside of the menu, you need to switch to controlled mode for the menu and pass it as `anchorRef` to `DropdownMenu`:
+
+```javascript live noinline
+// import {DropdownMenu, ActionList} from '@primer/react/drafts'
+const {DropdownMenu, ActionList} = drafts // ignore docs silliness; import like that ↑
+
+const Example = () => {
+  const [open, setOpen] = React.useState(false)
+  const anchorRef = React.createRef()
+
+  return (
+    <>
+      <Button ref={anchorRef} onClick={() => setOpen(!open)}>
+        {open ? 'Close Menu' : 'Open Menu'}
+      </Button>
+
+      <DropdownMenu open={open} onOpenChange={setOpen} anchorRef={anchorRef}>
+        <DropdownMenu.Overlay>
+          <ActionList>
+            <ActionList.Item selected={true}>Text</ActionList.Item>
+            <ActionList.Item>Number</ActionList.Item>
+            <ActionList.Item>Date</ActionList.Item>
+            <ActionList.Item>Iteration</ActionList.Item>
+          </ActionList>
+        </DropdownMenu.Overlay>
+      </DropdownMenu>
+    </>
+  )
+}
+
+render(<Example />)
+```
+
+### With Overlay Props
+
+```javascript live noinline
+// import {DropdownMenu, ActionList} from '@primer/react/drafts'
+const {DropdownMenu, ActionList} = drafts // ignore docs silliness; import like that ↑
+
+const fieldTypes = [
+  {icon: TypographyIcon, name: 'Text'},
+  {icon: NumberIcon, name: 'Number'},
+  {icon: CalendarIcon, name: 'Date'},
+  {icon: SingleSelectIcon, name: 'Single select'},
+  {icon: IterationsIcon, name: 'Iteration'}
+]
+
+const Example = () => {
+  const handleEscape = () => alert('you hit escape!')
+
+  const [selectedIndex, setSelectedIndex] = React.useState(1)
+  const selectedType = fieldTypes[selectedIndex]
+
+  return (
+    <DropdownMenu>
+      <DropdownMenu.Button aria-label="Select field type" leadingIcon={selectedType.icon}>
+        {selectedType.name}
+      </DropdownMenu.Button>
+      <DropdownMenu.Overlay width="medium" onEscape={handleEscape}>
+        <ActionList>
+          {fieldTypes.map((type, index) => (
+            <ActionList.Item key={index} selected={index === selectedIndex} onSelect={() => setSelectedIndex(index)}>
+              <type.icon /> {type.name}
+            </ActionList.Item>
+          ))}
+        </ActionList>
+      </DropdownMenu.Overlay>
+    </DropdownMenu>
+  )
+}
+
+render(<Example />)
+```
+
+### With a custom anchor
```

Comment:
anchor may be confusing for readers as it's used in a different context further up
```suggestion
### With a custom trigger
```

Reply:
We call this `anchor` because the menu still anchors on `DropdownMenu.Anchor`.